### DISCUSSION TITLE
Mode-specific audio offset

### DISCUSSION THREAD

**Discussion by nobbele** [ppy/osu#14833] [Ideas] [answered]
I think having different audio offsets for each mode would appreciated by a lot of multi-mode players. Personally, I feel like I need a different offset for taiko, mania and standard due to reading differences. I also know some players like Jakads use audio offset in mania but not other modes to align in-game notes with real life keyboard taps (disabling hitsounds with 0% effect volume).

**Comment by peppy** [comment, marked as answer]
The only way this would ever be implemented is as part of "settings profiles", which allow saving and restoring all settings game-wide.

**Reply by nobbele** [reply]
Yeah that's even more useful if it includes background dim and skins.
